### PR TITLE
On editing device, only show assigned alert rules

### PR DESCRIPTION
**Pull request by SourceDoctor**
now only Alert Rules will be listed which are assigned to the edited Device
In advance Alert Rules which are assigned to Device Groups, the Device is a Member of,
are listed now also

### CODE REVIEW

File: includes/html/print-alert-rules.php [hunk 1]
```diff
@@ -101,26 +101,45 @@
 
 echo '</select></td>';
 
-$query = 'FROM alert_rules';
-$where = '';
-$param = [];
+$query_device = 'select alert_device_map.rule_id as id from alert_device_map';
```
Comment (laf) [inline includes/html/print-alert-rules.php]: Can you UPPERCASE the query so it looks like it's a MySQL query please. I.e `SELECT alert_device_map.rule_id AS id FROM alert_device_map`

Comment (SourceDoctor) [reply includes/html/print-alert-rules.php]: @laf 
done

**Comment by SourceDoctor** [comment]
@laf 
something else to do?

**Comment by PipoCanaja** [comment]
@SourceDoctor , @laf , @kkrumm1 
For the Alert Rule tab of the device properties, I would expect to see all rules that are applicable to this device, which are : 
- global rules
- rules applicable to this device
- rules applicable to any device groups this device is in

For the moment in the PR, the implementation only shows "rules applicable to this device or to the device groups the device is in"

What do you think ?

**Comment by SourceDoctor** [comment]
@PipoCanaja why should there be all available rules visible?
That makes no sense to me.
If i take a look into Alert Rules on a specific Device, i expect to see only the rules which are assigned to this device. (directly or via device group matching)
So you can directly see for what you are alerted on the device you are on.

For seeing all available rules there exists Alert->Alert Rules.

**Comment by SourceDoctor** [comment]
@PipoCanaja 
you are absolute right "global rules" are missing.

I'll patch add it in. :-) thx for pointing to it.

**Comment by SourceDoctor** [comment]
@PipoCanaja 
fixed :-)

**Comment by SourceDoctor** [comment]
@PipoCanaja 
everything else to fix?

**Comment by PipoCanaja** [comment]
Tested OK. Thx @SourceDoctor

**Pull request merged**